features/step_definitions/auction_steps.rb

```diff
@@ -19,11 +19,37 @@
                FactoryGirl.create(:auction, :closed, :with_bidders, :single_bid)
              when 'needs evaluation'
                FactoryGirl.create(:auction, :with_bidders, :evaluation_needed)
+             when 'within simplified acquisition threshold'
+               FactoryGirl.create(:auction, :between_micropurchase_and_sat_threshold, :available)
+             when 'below the micropurchase threshold'
+               FactoryGirl.create(:auction, :below_micropurchase_threshold, :available)
              else
                fail "Unrecognized auction type: #{label}"
              end
 end
 
+Given(/^there is an auction with a starting price between the micropurchase threshold and simplified acquisition threshold$/) do
+  @auction = FactoryGirl.create(:auction, :between_micropurchase_and_sat_threshold, :available)
+end
+
+Then(/^I should see that the auction indicates it is for small business only$/) do
+  auction_div = find('a', text: @auction.title)
+    .find(:xpath, "..")
+    .find(:xpath, "..")
+    .find(:xpath, "..")
+
+  expect(auction_div).to have_content('Small-business only')
+end
+
+Then(/^I should not see that the auction indicates it is for small business only$/) do
+  auction_div = find('a', text: @auction.title)
+    .find(:xpath, "..")
+    .find(:xpath, "..")
+    .find(:xpath, "..")
```

Comment:
I went with this approach because it lets me select the entire outer div given an instance of `Auction`/`AuctionPresenter`. Our markup doesn't otherwise make those divs uniquely selectable.